website/source/resources.html.erb

```diff
@@ -0,0 +1,94 @@
+  <li><strong><a href="https://aws.amazon.com/quickstart/architecture/nomad/">HashiCorp Nomad on AWS</a> - AWS Quickstart Guide for Nomad</strong></li>
+  <li><strong><a href="https://github.com/hashicorp/nomad/tree/master/terraform">Nomad Sandbox Environment for AWS and Azure</a> - Easily Provision a Combined Nomad, Consul and Vault Cluster on AWS or Azure</strong></li>
+  <li><strong><a href="https://www.katacoda.com/hashicorp/scenarios/nomad-introduction">Introduction to Nomad (Katacoda)</a> - Interactive Introduction to Nomad</strong></li>
+  <li><strong><a href="https://katacoda.com/hashicorp/scenarios/playground">Nomad Playground (Katacoda)</a> - Online Nomad Playground Environment</strong></li>
+</ul>
+
+<h2>Integrations</h2>
+
+<ul>
+  <li><strong><a href="https://verizon.github.io/nelson/">Verizon Nelson</a> - Automated, Multi-region Container Deployment with HashiCorp Nomad</strong></li>
+  <li><strong><a href="https://github.com/elsevier-core-engineering/replicator">Elsevier Replicator</a> - Automated Cluster and Job Scaling For HashiCorp Nomad</strong></li>
+  <li><strong><a href="https://github.com/jrasell/levant">Levant</a> - An open source templating and deployment tool for HashiCorp Nomad jobs</strong></li>
+  <li><strong><a href="https://github.com/underarmour/libra">UnderArmour Libra</a> - A Nomad Auto Scaler</strong></li>
+  <li><strong><a href="https://help.spotinst.com/hc/en-us/articles/115005038289-Nomad-Container-Management">Nomad Container Management with SpotInst</a> - Nomad Integration for SpotInst Elastigroup</strong></li>
+  <li><strong><a href="https://github.com/jenkinsci/nomad-plugin">Jenkins Nomad Plugin</a> - Nomad Cloud Plugin for Jenkins</strong></li>
+  <li><strong>OpenFaaS (<a href="https://registry.terraform.io/modules/nicholasjackson/open-faas-nomad/aws/0.4.0">Terraform</a> | <a href="https://github.com/openfaas/faas">GitHub</a>) - Nomad Integration for OpenFaas (Functions as a Service)</strong></li>
+  <li><strong><a href="https://github.com/ValFadeev/rundeck-nomad-plugin">Rundeck Plugin for Nomad</a> - A Rundeck Plugin for Running Jobs on a Nomad Cluster</strong></li>
+</ul>
+
+<h2>Other</h2>
+
+<ul>
+  <li><strong><a href="https://github.com/jippi/awesome-nomad">jippi/awesome-nomad</a> - A Curated List of Nomad Tools</strong></li>
+  <li><strong><a href="https://github.com/jrasell/nomadfiles">jrasell/nomadfiles</a> - A Collection of Nomad Job Files for Deploying Applications</strong></li>
+</ul>
+
+<h2>Trusted By</h2>
+
+<ul>
+  <li><strong>Barclays</strong></li>
+  <li><strong>CircleCI</strong></li>
+  <li><strong>Citadel</strong></li>
+  <li><strong>Deluxe Entertainment</strong></li>
+  <li><strong>Elsevier</strong></li>
+  <li><strong>Jet.com</strong></li>
+  <li><strong>Pagerduty</strong></li>
+  <li><strong>SAP Ariba</strong></li>
+  <li><strong>Spaceflight Industries</strong></li>
+  <li><strong>SpotInst</strong></li>
+  <li><strong>UnderArmour</strong></li>
+  <li><strong>Verizon</strong></li>
```

Comment:
can add Bownty.com and SeatGeek as well :)

Reply:
Send a PR our way!

Reply:
Added!